deploy-argocd.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -e
+
```

Comment:
it would be great to add some comments to the top of the script and explain its purpose.

Reply:
Updated, please check

Reply:
The description is misleading and needs to be updated.
The script does not deploy ArgoCD as the Terraform configuration takes care of that.